packages/app/src/settings/store/schemas/settings.schema.ts

```diff
@@ -13,12 +13,26 @@ import translationConfig from "App/translations.config.json"
 
 const generateApplicationId = (): string => {
   const maxApplicationIdLength = 16
-  const uniqueValue = getMAC().replace(/:/g, "").slice(-maxApplicationIdLength)
+
+  let mac = ""
+  try {
+    mac = getMAC()
+  } catch (ex) {
+    console.log(ex)
```

Comment:
Is this intentional? Maybe we should do something else with errors. Or maybe i am just being picky :D

Reply:
Yeap, it's intentional, but I agree console.log might not be good idea, but what else we can do about it?

Reply:
@dkarski @lkowalczyk87 do you have any ideas? I've updated [Topics to discuss](https://appnroll.atlassian.net/wiki/spaces/CP/pages/1856471889/Topics+to+discuss) with this issue.

Reply:
Let's don't block this PR and talk about it in the near front-end meeting 👍